src/components/Slider/Slider.module.scss

```diff
@@ -0,0 +1,78 @@
+.RootContainer {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+.LightRootContainer {
+  color: black;
+}
+
+.SliderRoot {
+  position: relative;
+  display: flex;
+  align-items: center;
+  width: 18.75rem;
+  background-color: #37273f;
```

Comment:
Could we add the colours used to the colors variables file please 🙂 Keep them in line with what they are called in the figma designs

<img width="1424" alt="Screenshot 2022-11-30 at 11 26 07" src="https://user-images.githubusercontent.com/39112648/204784254-5867347e-ae50-4c37-8325-eb317a17472b.png">

<img width="1424" alt="Screenshot 2022-11-30 at 11 28 14" src="https://user-images.githubusercontent.com/39112648/204784972-c4c959eb-2d30-4d5c-addb-018beba0fc2f.png">

Reply:
I have added them in, but if you could check that the naming convention is what you were expecting that would be awesome.